config/src/config/quorum_store_config.rs

```diff
@@ -9,23 +9,26 @@ use serde::{Deserialize, Serialize};
 #[serde(default, deny_unknown_fields)]
 pub struct QuorumStoreBackPressureConfig {
     pub backlog_txn_limit_count: u64,
+    pub backlog_batch_limit_count: u64,
     pub decrease_duration_ms: u64,
     pub increase_duration_ms: u64,
     pub decrease_fraction: f64,
-    pub dynamic_min_batch_count: u64,
-    pub dynamic_max_batch_count: u64,
+    pub dynamic_min_txn_per_s: u64,
+    pub dynamic_max_txn_per_s: u64,
 }
 
 impl Default for QuorumStoreBackPressureConfig {
     fn default() -> QuorumStoreBackPressureConfig {
         QuorumStoreBackPressureConfig {
             // QS will be backpressured if the remaining total txns is more than this number
             backlog_txn_limit_count: MAX_SENDING_BLOCK_TXNS_QUORUM_STORE_OVERRIDE * 4,
+            // QS will create batches immediately until this number is reached
+            backlog_batch_limit_count: 80,
```

Comment:
this is a global limit? why only 80? 4 blocks per second is status quo, which is >400, because latency is above 1s.

So maybe we should use 400 here? (maybe even a 1000)

thinking outloud, or maybe we should have a map of backlog_batch_limit_count to batch_generation_min_interval_ms, based on the backlog:
below 80, batch_generation_min_interval_ms is 0
below 200, batch_generation_min_interval_ms is 50,
below 400, batch_generation_min_interval_ms is 100,
below 1000, batch_generation_min_interval_ms is 200,
above 1000, batch_generation_min_interval_ms is 250?

Reply:
Yeah unfortunately this has to be tuned per network at the moment -- 80 makes sense for 20 validator networks which is what is mostly in Forge.

For now I'd like to start with a single value instead of a map/function so it's a bit easier to understand.

Reply:
We'll need an override in `preview`

Reply:
actually, we always create batch on 250, even if we are in backlog_batch_limit_count, so probably not need to complicate with levels. 

But I would still put it in the range of 400-1000 ?